README.md

```diff
@@ -112,80 +112,37 @@ Basic OpenShift configuration:
 
 ```yaml
 name: openshift
-location: eastus
+location: $AZURE_REGION
 properties:
   openShiftVersion: v3.10
-  publicHostname: openshift.$RESOURCEGROUP.$DNS_DOMAIN
+  publicHostname: $RESOURCEGROUP.$AZURE_REGION.cloudapp.azure.com
```

Comment:
Does it make sense to keep the `$DNS_DOMAIN` here?

Reply:
Probably not as long as we invalidate everything except [`.cloudapp.azure.com`](https://github.com/openshift/openshift-azure/pull/535).

Reply:
Thanks @kwoodson this was a typo, it should have read `fqdn: $RESOURCEGROUP.$AZURE_REGION.cloudapp.azure.com`.  Fixed